packages/fury-adapter-remote/test/fury-test.js

```diff
@@ -258,11 +275,12 @@ describe('Adapter works with Fury interface', () => {
       }));
     });
 
-    it('parses blueprint with mediaType', () => {
+    it('parses blueprint with mediaType', (done) => {
       fury.parse({ source: blueprintSource, mediaType: 'text/vnd.apiblueprint' }, (err, result) => {
         expect(result).to.be.undefined;
         expect(err).to.be.instanceof(Error);
-        expect(err).to.have.property('message', 'Request failed with status code 415');
+        expect(err).to.have.property('message', 'Request failed with status code 404');
```

Comment:
This is in a test `with invalid endpoint` (one level up in describe) and thus the API returns 404 for hitting an unknown path.